### PR TITLE
removed old adapter au_sa.js

### PR DESCRIPTION
**Pull request by majesticio**
adapter has been replaced by southaustralia.js.

### CODE REVIEW

**Comment by majesticio** [comment]
Do we need to add this to the change log?

File: src/adapters/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```
Comment (russbiggs) [inline src/adapters/CHANGELOG.md:6]: We're not going to follow semantic versioning so remove this

File: src/adapters/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- `Added` for new features.
+- `Changed` for changes in existing functionality.
+- `Deprecated` for soon-to-be removed features.
+- `Removed` for now removed features.
+- `Fixed` for any bug fixes.
+- `Security`
+
+## [unversioned]
+
+## [0.0.1] - 04/11/2023
```
Comment (russbiggs) [inline src/adapters/CHANGELOG.md:18]: Just used the date since we arent following a versioning scheme

File: src/adapters/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
```
Comment (russbiggs) [inline src/adapters/CHANGELOG.md:3]: This changelog will only be relevant for adapters, so maybe language like "Changes to adapters wil lbe documented in this file"

File: src/adapters/CHANGELOG.md [hunk 4]
```diff
@@ -1,23 +1,9 @@
...
-- `Changed` for changes in existing functionality.
-- `Deprecated` for soon-to-be removed features.
-- `Removed` for now removed features.
-- `Fixed` for any bug fixes.
-- `Security`
-
-## [unversioned]
-
-## [0.0.1] - 04/11/2023
+Changes to adapters wil lbe documented in this file
```
Comment (russbiggs) [inline src/adapters/CHANGELOG.md:3]: typo at "wil lbe"

**Review by russbiggs — CHANGES_REQUESTED** [review]

**Comment by russbiggs** [comment]
@majesticio whats the status on this PR?

**Comment by majesticio** [comment]
I'm closing this PR as theses changes have already been implemented.

**Pull request closed without merging**